### PR TITLE
Make elasticsearch clusterName configurable

### PR DESCRIPTION
**Pull request by PashCracken**
Since we install several elasticsearch instances for different purposes in our Kubernetes namespace we need to be able to customize the elasticsearch cluster name. We can currently do this in the zeebe-cluster helm chart, but not in this one

### CODE REVIEW

**Comment by salaboy** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **salaboy**

If they are not already assigned, you can assign the PR to them by writing `/assign @salaboy` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=zeebe-io%2Fzeebe-operate-helm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/zeebe-io/zeebe-operate-helm/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by salaboy** [comment]
Hi @PashCracken. Thanks for your PR.

I'm waiting for a [zeebe-io](https://github.com/orgs/zeebe-io/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/zeebe-io/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=zeebe-io%2Fzeebe-operate-helm).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by salaboy** [comment]
@PashCracken did you tested these changes? I was trying to do something similar but I couldn't get the elasticsearch cluster name changed.. 
In this case is just for the Operate chart to connect to a different name right?

**Comment by PashCracken** [comment]
@salaboy Yes. I've tested it. We are currently running Operator using these changes. Our elasticsearch cluster is named "zeebe-elasticsearch". This one PR is just for Operate to connect to a cluster with another name. Here you see parts of the values file we use. This all works fine now

```
global:
  zeebe: zeebe
  elasticsearch:
    host: zeebe-elasticsearch-master
    clusterName: zeebe-elasticsearch
zeebe-cluster: 
  fullnameOverride: zeebe
  kibana:
    enabled: true
    elasticsearchHosts: http://zeebe-elasticsearch-master:9200
    labels:
      app.kubernetes.io/part-of: zeebe
    ingress:
      enabled: true
      annotations:
        kubernetes.io/ingress.class: nginx
      hosts:
        - zeebe-kibana.kube.firmglobal.com
  elasticsearch:
    clusterName: zeebe-elasticsearch
    labels:
      app.kubernetes.io/part-of: zeebe
```

**Review by salaboy — APPROVED** [review]

**Comment by salaboy** [comment]
@PashCracken great I will merge and release tomorrow morning my time

**Comment by PashCracken** [comment]
@salaboy When will the new Helm chart version be published? I still only see 0.0.12

**Comment by salaboy** [comment]
@PashCracken done: https://github.com/zeebe-io/zeebe-full-helm/releases/tag/v0.0.20 included in .20 and operate .13

**Pull request merged**
